development/prow-installer/pkg/cluster/wrapped.go

```diff
@@ -55,17 +56,36 @@ func (caw *APIWrapper) Create(ctx context.Context, clusterConfig Cluster) error
 			NodePools:      nodePools,
 		}}
 
-	createResponse, err := caw.ClusterService.Create(caw.ProjectID, clusterConfig.Location, ccRequest).Context(ctx).Do()
+	_, err := caw.ClusterService.Create(caw.ProjectID, clusterConfig.Location, ccRequest).Context(ctx).Do()
```

Comment:
Wondering if there could be a case where the ProjectID comes from the cluster config.